lib/eventsource-polyfill.js

```diff
@@ -1,5 +1,6 @@
 var EventSource = require('./eventsource')
 if (typeof window === 'object') {
   window.EventSource = window.EventSource || EventSource
+} else if (typeof global === 'object') {
+  global.EventSource = global.EventSource || EventSource
```

Comment:
I'm hoping to not have to do this in a module that's designed for both node & browser:
```js
const EventSource = (typeof window === 'object' && window.EventSource)
  ? window.EventSource
  : require('eventsource');
```

Are you ok with something like this for `eventsource-polyfill.js`:
```js
var EventSource = require('./eventsource')
if (typeof window === 'object') {
  if (!window.EventSource) window.EventSource = EventSource
  module.exports = window.EventSource;
} else {
  module.exports = EventSource;
}
```
That way, eventsource-polyfill becomes a commonjs module (that also polyfills the browser).